students/rick_halsell/lesson03/list_lab.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
+import contextlib
+# Series 1
+
+def fruitselectionfunc():
+    global FruitSelection
+    FruitSelection = input('2. Select an item from the list of fruit based on number > ')
+    return FruitSelection
```

Comment:
Not exactly sure what's the intention here.  This isn't returning the selected fruit to the user.  Also, recommend using len() so you can show the user the length of the existing list.